org.testeditor.web.backend.persistence/src/test/java/org/testeditor/web/backend/persistence/DocumentProviderTest.xtend

```diff
@@ -18,12 +21,124 @@ class DocumentProviderTest extends AbstractGitTest {
 
 	@Test
 	def void createCommitsNewFile() {
+		// given
+		val numberOfCommitsBefore = git.log.call.size
+
 		// when
 		documentProvider.create('a.txt', 'test')
 
 		// then
-		val commits = git.log.call
-		commits.assertSize(2)
+		val numberOfCommitsAfter = git.log.call.size
+		numberOfCommitsAfter.assertEquals(numberOfCommitsBefore + 1)
```

Comment:
why not use the assertSingleCommit function you use in other test methods?

Reply:
This was the only test case that was already present (I just modified it), and somehow I must have missed it when I introduced the helper method during a later refactoring.